pkg/keyctl/key.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 Jesse Sipprell. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build linux
```

Comment:
Constraints here.

Reply:
do we need constraints in pkg/keyctl/perm.go? :thought_balloon: